camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -236,6 +290,26 @@ def do_upload(self):
             rv = rv and self.check_set_camera_info(response)
         return rv
 
+    def ask_yes_no_question(self, question, default_yes=True):
```

Comment:
this needs at least a @classmethod decorator as it is not using "self".
however i would rather move it outside of the class and have it somewhere as a helper method.